### ISSUE TITLE
window.cordova is undefined after upgrade to 1.0.0

### ISSUE THREAD

**Issue by joggeli34** [getsentry/sentry-cordova#220]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
Angular/Ionic 11.2.10 with sentry-cordova 1.0.0

### Steps to Reproduce

When updating the dependency from "sentry-cordova" 0.17.0 to 1.0.0, then `window.cordova` stayes undefined. So I can't use other plugins like "InAppBrowser"

It only happens on android. ios is still working without a problem.

### Expected Result
`window.cordova` should stay defined.

### Actual Result

`window.cordova` is undefined

**Comment by jennmueng** [comment]
Hmm, this is unusual, can you provide a reproduction @joggeli34?

**Comment by joggeli34** [comment]
I created a reproducer project (with capacitor, ionic, angular)

The easiest way to run is with installed ionic-cli:

```
npm install
ionic cap sync android
ionic cap open android
```

And then run in android studio.

The project as provided here has included `sentry-cordova: 0.17.0`, then it shows `Has Cordova` on the app-screen. When changing the version of `sentry-cordova` to 1.0.0, it shows `no cordova`

The code to check is simply:

```ts
  public hasWindowCordova(): boolean {
    // @ts-ignore
    console.log(!window.cordova);
    // @ts-ignore
    return !!window.cordova;
  }
```

with the template:
```html
<h2 *ngIf="hasWindowCordova()">Has Cordova</h2>
<h2 *ngIf="!hasWindowCordova()">No Cordova</h2>
```

It's just a simple project generated with ionic-cli with added sentry-cordova library (and removed some of the not interessting code)

I hope this helps to reproduce it.

Regards


[test_missing_cordova.zip](https://github.com/getsentry/sentry-cordova/files/6376804/test_missing_cordova.zip)
